group-work.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: episode
+title: How to organize a group's work
+teaching: 15
+exercises: 15
+questions:
+  - Should I start a repo under my account or open a new organization?
+  - Should I add everybody as collaborator?
+  - GitHub vs GitLab?
+  - What permissions to choose?
+  - Pull requests vs direct commits for types of projects
+  - Small vs large changes
+---
+
+# How to organize a group's work
+
+---
+
```

Comment:
Here, we should add something explaining "do you want to work together?  Do you?  What are the benefits?  How much of our individual work is caused by not having a way to collaborate?  If we could collaborate, would it improve the quality of our work.  Do we want to?"

This is to frame the answer to some of the things below.  You know I am very practical on PRs and how it can only work if we get people interested enough in each other's work.

Reply:
I will add a sentence.